dahee/app.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express')
+const cors = require('cors')
+const app = express()
+const dotenv = require('dotenv')
+
+dotenv.config()
+
+app.use(cors())
+app.use(express.json())
+//app.use(morgan("combined"))
+
+const { DataSource } = require('typeorm')
+
+const myDataSource = new DataSource({
+    type: process.env.DB_CONNECTION,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE
+})
+
+myDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    })
+
+app.listen(3000, () => {
+    console.log('port is listening')
+})
```

Comment:
```suggestion
app.listen(PORT, () => {
    console.log('port is listening')
})
```
port number는 소스코드에 하드코딩 하기보다, `.env`에 정의하여 소스코드에서는 환경변수에서 불러와 사용할 수 있도록 해주세요.

```javascript
. . .
const PORT = process.env.PORT
. . .
```